internal/datatransfer/model/filesync.go

```diff
@@ -0,0 +1,15 @@
+package model
```

Comment:
but it was all part of datatransfer package, no?

Reply:
Correct. It's because of import cycling issues between datatransfer and metrics:
```
package command-line-arguments
	imports github.com/project-flotta/flotta-device-worker/internal/metrics
	imports github.com/project-flotta/flotta-device-worker/internal/datatransfer
	imports github.com/project-flotta/flotta-device-worker/internal/datatransfer: import cycle not allowed

```